src/main/java/it/gov/pagopa/reward/dto/mapper/RewardTransaction2SynchronousTransactionResponseDTOMapper.java

```diff
@@ -19,6 +19,9 @@ public SynchronousTransactionResponseDTO apply(String trxId, String initiativeId
         out.setChannel(rewardTransactionDTO.getChannel());
         out.setInitiativeId(initiativeId);
         out.setUserId(rewardTransactionDTO.getUserId());
+        out.setOperationType(rewardTransactionDTO.getOperationTypeTranscoded());
+        out.setAmount(rewardTransactionDTO.getAmountCents());
```

Comment:
let's return amount not amountCents

every currency in other fields returned are euro (effectiveAmount, reward.accruedReward)